examples/sandbox/index.ts

```diff
@@ -585,25 +585,25 @@ $getXpubs.on("click", async (e) => {
   })[0];
 
   const result = await wallet.getPublicKeys([
```

Comment:
This will need to be programmatic in sandbox - we can't just have a magic xpubs button getting xpubs for all coins - the way the Ledger works is only one app can be connected at a time, so only the call to the chain matching the connected app would work